src/main/python/msmarco/convert_trec_to_msmarco_run.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Anserini: A Lucene toolkit for replicable information retrieval research
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Converts a TREC run file to a MS MARCO-formatted run file.')
+    parser.add_argument('--input_run', required=True, default='', help='TREC-formatted run file')
```

Comment:
how about just `input` and `output` to be consistent with args from other scripts?

Reply:
Sure